### ISSUE TITLE
[Shaders]: Z-fighting (only on some items) in inventory with shaders

### ISSUE THREAD

**Issue by Darek505** [GTNewHorizons/Angelica#696]
### Your GTNH Discord Username

darek505

### Mod Version

1.0.0-beta14

### Bug Report

https://github.com/user-attachments/assets/4e1b11f5-58b9-4b34-b73e-d6456957a3fa

### Java Version

Java 8

### Graphics Card Vendor

NVidia

### Shaders

Yes: Please put relevant information about shaderpack in the report(what shaderpack, any special settings of note, etc)

### MCPF or other features disabled by default

No: Go ahead and create an issue

### Mod List or GTNH Pack Version

[✔] +unimixins-all-1.7.10-0.1.18
[✔] angelica-1.0.0-beta14
[✔] gtnhlib-0.5.18

### Final Checklist

- [X] I have searched the issues and haven't found a similar issue.
- [X] I have read the known incompatibilities and this is not related to one of those.
- [X] I am running an officially released version.  (Or, if I've compiled it myself I plan to fix the issue)

**Comment by Cleptomania** [comment]
Have you verified if this only happens when shaders are enabled?

**Comment by Darek505** [comment]
Yes, this is quite difficult to reproduce, as it does not always work, this *can* happen the first time you launch the shader, if after that you turn off the shaders, and then turn them on again, this error *should* go away

**Comment by Cleptomania** [comment]
I've seen it happen before, but never actually looked into re-producing it or what circumstances it happened under

**Comment by mchernykh** [comment]
I have the same problem
Java 21
2.7 beta 3
Nvidia 1070

**Comment by MalTeeez** [comment]
Tested a bit, can't reproduce on latest: 

<img width="762" height="435" alt="Image" src="https://github.com/user-attachments/assets/5b70e6b6-a215-4a16-8a5e-40a2f45b19c3" />
